src/main/java/com/google/cloud/tools/maven/CloudSdkMojo.java

```diff
@@ -47,22 +44,9 @@
 
   @Override
   public void execute() throws MojoExecutionException, MojoFailureException {
-    // ensure that the Cloud SDK is available
-    if (cloudSdkPath == null) {
-      try {
-        Path sdkPath = PathResolver.INSTANCE.getCloudSdkPath();
-        if (sdkPath != null) {
-          cloudSdkPath = PathResolver.INSTANCE.getCloudSdkPath().toFile();
-        }
-      } catch (FileNotFoundException e) {
-        throw new MojoFailureException(
-            "Cloud SDK Path was not provided, and cannot be automatically detected.", e);
-      }
-    }
-    if (processRunner == null) {
-      this.cloudSdk = new CloudSdk(cloudSdkPath);
-    } else {
-      this.cloudSdk = new CloudSdk(cloudSdkPath, processRunner);
-    }
+    this.cloudSdk = new CloudSdk.Builder()
```

Comment:
yep, very recently https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/54